src/tplink_omada_client/devices.py

```diff
@@ -97,6 +98,11 @@ def firmware_version(self) -> str:
         """Firmware version of the device"""
         return self._data["firmwareVersion"]
 
+    @property
+    def led_setting(self) -> LedSetting:
```

Comment:
Unfortunately, I don't think this property can go here. The ledSetting value isn't available when listing all devices. So I think the property should be in the classes `OmadaSwitch', 'OmadaAccessPoint' and 'OmadaGateway', or possibly an intermediate class in the hierarchy, as they all appear to share some data.